packages/account/src/Sections/Security/ConnectedApps/connected-apps-article.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { localize, Localize } from '@deriv/translations';
+import AccountArticle from 'Components/article';
+
```

Comment:
can't we use the default name of the component? Isn't more readable?
`import Article from 'Components/article';`

Reply:
I saw existing component using the same Name which is `AccountArticle` should i rename those also?

Reply:
@suisin It is exported as AccountArticle from index file. No need to change anything. Thanks